activities/FoodChain.activity/playgame.js

```diff
@@ -10,14 +10,25 @@ FoodChain.playLevels = [
 
 // Key to use
 FoodChain.playKeys = [
-	{ key: 75, heading: 0, dx: 1, dy: 0 },
-	{ key: 107, heading: 0, dx: 1, dy: 0 },
-	{ key: 73, heading: 90, dx: 0, dy: -1 },
-	{ key: 105, heading: 90, dx: 0, dy: -1 },
-	{ key: 74, heading: 0, dx: 1, dy: 0 },
-	{ key: 106, heading: 180, dx: -1, dy: 0 },
-	{ key: 76, heading: 270, dx: 0, dy: 1 },
-	{ key: 108, heading: 270, dx: 0, dy: 1 }
+	//commenting out the K key-codes and replacing 
```

Comment:
@ricknjacky , you don't need to add these comments.

Reply:
@prakashujjwal1010 I added them, thinking some time in the future they might help.I will remove the comments and commit again :) thanks for the suggestion